ensembldl.sh

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/env bash
+
+# safety first
+set \
+  -o errexit \
+  -o pipefail \
+  -o noglob \
+  -o nounset \
+
+app=$(basename "$0" .sh)
+
+version=$(git describe --always --long --dirty 2> /dev/null) ||
+  version="0.1.0"
+
+# get utilities
+# shellcheck source=util.sh
+source "$(dirname "$0")"/util.sh
+
+# -----------------------------------------------------------------------------
+# usage
+# -----------------------------------------------------------------------------
+
+function usage { cat << EOF
+$app $version
+
+USAGE
+
+  $app [options] [--] prefix dataset...
+
+DESCRIPTION
+
+  download NCBI data set
```

Comment:
This shouldn't be **NCBI** anymore, should it?

Reply:
done